docs-ref-autogen/TOC.md

```diff
@@ -12,6 +12,7 @@
 ## [Output formats](../docs-ref-conceptual/format-output-azure-cli.md)
 ## [Query](../docs-ref-conceptual/query-azure-cli.md)
 ## [Subscriptions](../docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md)
+## [Cloud Sets](../docs-ref-conceptual/manage-cloud-sets-azure-cli.md)

```

Comment:
I would suggest just 'Clouds' instead of 'Cloud Sets'. Same everywhere else in this PR.
Was there a reason for the name 'Cloud Sets'?